docs/spec/tendermint-fork-cases/TendermintAcc3.tla

```diff
@@ -438,10 +426,49 @@ AgreementOrEquivocationOrAmnesiaOld ==
 AgreementOrEquivocationOrAmnesia ==
     \/ Agreement
     \/ \E Detectable \in SUBSET Faulty:
-        /\ Cardinality(Detectable) >= THRESHOLD1
+        /\ Cardinality(Detectable) = THRESHOLD1
         /\ \A p \in Detectable:
             EquivocationBy(p) \/ AmnesiaBy(p)
+ 
+\* This property is violated. You can check it to see how amnesic behavior
+\* appears in the evidence variable.
+NoAmnesia ==
+    \A p \in Faulty: ~AmnesiaBy(p)
+
+\* This property is violated. You can check it to see an example of equivocation.
+NoEquivocation ==
+    \A p \in Faulty: ~EquivocationBy(p)
+
+\* This property is violated. You can check it to see an example of agreement.
+\* It is not exactly ~Agreement, as we do not want to see the states where
+\* decision[p] = NilValue
+NoAgreement ==
+  \A p, q \in Corr:
+    (p /= q /\ decision[p] /= NilValue /\ decision[q] /= NilValue)
+        => decision[p] /= decision[q]
+ 
+\* We expect this property to be violated. However, the absence of amnesia
+\* is a tough constraint for Apalache. It has not reported a counterexample
+\* for n=4,f=2, length <= 5.
+NoAmnesiaButAgreementOrEquivocation ==
```

Comment:
the fomula is an implication in disguise. Perhaps rename to`NoAmnesiaImpliesAgreementOrEquivocation`?

Reply:
The question is interesting. Perhaps we can massage the temporal property a bit to help Apalache? Or do you see fundamental issues here?

Reply:
Fixed.

Reply:
Actually, we can restrict apalache to execute a specific sequence of actions. Then it can instantiate the messages and the states to show an execution that follows this trace.